internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -196,6 +198,16 @@ func deviceHandler(writer http.ResponseWriter, request *http.Request) {
 	handler.processAsyncRequest(writer, request)
 }
 
+func convertToBase64(val []byte) string {
+	// Decode the CBOR format
+	var h codec.Handle = new(codec.CborHandle)
```

Comment:
Can you explain why we should do the CBOR processing before converting image byte streams to BASE64 directly?
Is there any problems without CBOR processing?

Reply:
All binary data being transferred to EdgeX in in CBOR format, thus I had put a decoder. But you are right, I need to test it without the decoder too. I will do that and remove it if it is unnecessary.

Reply:
It worked, I removed that part.